templates/README.md

```diff
@@ -0,0 +1,6 @@
+# Steps
+1) Navigate to the correct directory `cd k8cher\templates`
+2) Install the template `dotnet new -i .\K8cherService\`
+3) Create a new service named MyService `dotnet new K8cherService -n MyService`
```

Comment:
I would suggest adding a new step to clarify that the last step should be done in the src directory:
3) Navigate to the src directory `cd ..\src`
4) Create a new service named MyService `dotnet new K8cherService -n MyService`